dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/LocalFileTreeTablePresenter.java

```diff
@@ -313,7 +318,7 @@ else if (Guard.isNullOrEmpty(values)) {
 
         } catch (final Exception e) {
             LOG.error("Failed to transfer data to black pearl", e);
-            ds3Common.getDeepStorageBrowserPresenter().logText("Failed to transfer data to black pearl: " + e, LogType.ERROR);
+            loggingService.logMessage("Failed to transfer data to black pearl: " + e, LogType.ERROR);
```

Comment:
Haven't seen enough to know what % is out of resource or internationalizable, but is it worth forcing all error messages?